webapp/src/actions/index.js

```diff
@@ -151,3 +151,44 @@ export function getUnreads() {
         return {data};
     };
 }
+
+const GITHUB_USER_GET_TIMEOUT_MILLISECONDS = 1000 * 60 * 60; // 1 hour
+
+export function getGitHubUser(userID) {
+    return async (dispatch, getState) => {
+        if (!userID) {
+            return {};
+        }
+
+        const user = getState()['plugins-github'].githubUsers[userID];
+        if (user && user.last_try && Date.now() - user.last_try < GITHUB_USER_GET_TIMEOUT_MILLISECONDS) {
```

Comment:
Some basic backoff logic so we don't spam the plugin every time the popover is re-opened for an unconnected user.